.github/issue-close-app.yml

```diff
@@ -0,0 +1,30 @@
+# Comment that will be sent if an issue is judged to be closed.
```

Comment:
I would delete this comment, and instead explain what the large-scale goal of this file is, because if I came across this later I would have no idea. Perhaps:
`# Config for a bot that automatically closes issues if they do not conform to some simple rules`

Thoughts?